index.html

```diff
@@ -180,7 +180,24 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
     <p class="setup-similar-label">Петр Петров</p>
   </div>
 </template>
-
+<script>
+  'use strict';
+  var fireballSize = 22;
+  var getFireballSpeed = function (left) {
+    return left ? 5 : 2;
+  };
+  var wizardSpeed = 3;
+  var wizardWidth = 70;
+  var getWizardHeight = function () {
+    return 1.337 * wizardWidth;
+  };
+  var getWizardX = function (width) {
+    return (width * 0.5) - wizardWidth;
```

Comment:
```suggestion
    return (width * 0.5) - wizardWidth * 0.5;
```
ведь точка X должна находиться в центре мага по ширине

Reply:
И 0.5 можно и за скобки вынести